// locutus/golang/path/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.Match = exports.Join = exports.IsAbs = exports.Ext = exports.Dir = exports.Clean = exports.Base = void 0;
var Base_ts_1 = require("./Base.js");
Object.defineProperty(exports, "Base", { enumerable: true, get: function () { return Base_ts_1.Base; } });
var Clean_ts_1 = require("./Clean.js");
Object.defineProperty(exports, "Clean", { enumerable: true, get: function () { return Clean_ts_1.Clean; } });
var Dir_ts_1 = require("./Dir.js");
Object.defineProperty(exports, "Dir", { enumerable: true, get: function () { return Dir_ts_1.Dir; } });
var Ext_ts_1 = require("./Ext.js");
Object.defineProperty(exports, "Ext", { enumerable: true, get: function () { return Ext_ts_1.Ext; } });
var IsAbs_ts_1 = require("./IsAbs.js");
Object.defineProperty(exports, "IsAbs", { enumerable: true, get: function () { return IsAbs_ts_1.IsAbs; } });
var Join_ts_1 = require("./Join.js");
Object.defineProperty(exports, "Join", { enumerable: true, get: function () { return Join_ts_1.Join; } });
var Match_ts_1 = require("./Match.js");
Object.defineProperty(exports, "Match", { enumerable: true, get: function () { return Match_ts_1.Match; } });
