// locutus/esm/php/var/index.js
export { boolval } from "./boolval.js";
export { doubleval } from "./doubleval.js";
export { empty } from "./empty.js";
export { floatval } from "./floatval.js";
export { gettype } from "./gettype.js";
export { intval } from "./intval.js";
export { is_array } from "./is_array.js";
export { is_binary } from "./is_binary.js";
export { is_bool } from "./is_bool.js";
export { is_buffer } from "./is_buffer.js";
export { is_callable } from "./is_callable.js";
export { is_double } from "./is_double.js";
export { is_float } from "./is_float.js";
export { is_int } from "./is_int.js";
export { is_integer } from "./is_integer.js";
export { is_long } from "./is_long.js";
export { is_null } from "./is_null.js";
export { is_numeric } from "./is_numeric.js";
export { is_object } from "./is_object.js";
export { is_real } from "./is_real.js";
export { is_scalar } from "./is_scalar.js";
export { is_string } from "./is_string.js";
export { is_unicode } from "./is_unicode.js";
export { isset } from "./isset.js";
export { print_r } from "./print_r.js";
export { serialize } from "./serialize.js";
export { strval } from "./strval.js";
export { unserialize } from "./unserialize.js";
export { var_dump } from "./var_dump.js";
export { var_export } from "./var_export.js";
