// locutus/esm/php/_helpers/index.js
export { _bc } from "./_bc.js";
export { _phpCastString } from "./_phpCastString.js";
export { _php_cast_float } from "./_php_cast_float.js";
export { _php_cast_int } from "./_php_cast_int.js";
export { assertIsNumericLike } from "./_phpTypes.js";
export { assertIsObjectLike } from "./_phpTypes.js";
export { assertIsPhpAssocObject } from "./_phpTypes.js";
export { assertIsPhpCallable } from "./_phpTypes.js";
export { assertIsPhpKey } from "./_phpTypes.js";
export { assertIsPhpList } from "./_phpTypes.js";
export { ensurePhpRuntimeState } from "./_phpRuntimeState.js";
export { entriesOfPhpAssoc } from "./_phpTypes.js";
export { getArrayLikeLength } from "./_arrayPointers.js";
export { getCtypePattern } from "./_ctypePattern.js";
export { getEntryAtCursor } from "./_arrayPointers.js";
export { getPhpGlobalCallable } from "./_phpRuntimeState.js";
export { getPhpGlobalEntry } from "./_phpRuntimeState.js";
export { getPhpGlobalScope } from "./_phpRuntimeState.js";
export { getPhpLocaleGroup } from "./_phpRuntimeState.js";
export { getPhpObjectEntry } from "./_phpRuntimeState.js";
export { getPhpRuntimeBoolean } from "./_phpRuntimeState.js";
export { getPhpRuntimeEntry } from "./_phpRuntimeState.js";
export { getPhpRuntimeNumber } from "./_phpRuntimeState.js";
export { getPhpRuntimeString } from "./_phpRuntimeState.js";
export { getPointerState } from "./_arrayPointers.js";
export { isNumericLike } from "./_phpTypes.js";
export { isObjectLike } from "./_phpTypes.js";
export { isPhpArrayObject } from "./_phpTypes.js";
export { isPhpAssocObject } from "./_phpTypes.js";
export { isPhpCallable } from "./_phpTypes.js";
export { isPhpCallableDescriptor } from "./_phpTypes.js";
export { isPhpKey } from "./_phpTypes.js";
export { isPhpList } from "./_phpTypes.js";
export { isPhpNullish } from "./_phpTypes.js";
export { isPhpScalar } from "./_phpTypes.js";
export { resolveNumericComparator } from "./_callbackResolver.js";
export { resolvePhpCallable } from "./_callbackResolver.js";
export { setPhpGlobalEntry } from "./_phpRuntimeState.js";
export { setPhpLocaleDefault } from "./_phpRuntimeState.js";
export { setPhpObjectEntry } from "./_phpRuntimeState.js";
export { setPhpRuntimeEntry } from "./_phpRuntimeState.js";
export { toPhpArrayObject } from "./_phpTypes.js";
