// locutus/python/math/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.trunc = exports.sqrt = exports.pow = exports.log2 = exports.log10 = exports.log = exports.lcm = exports.isnan = exports.isinf = exports.isfinite = exports.isclose = exports.gcd = exports.fsum = exports.floor = exports.factorial = exports.fabs = exports.exp = exports.dist = exports.comb = exports.ceil = void 0;
var ceil_ts_1 = require("./ceil.js");
Object.defineProperty(exports, "ceil", { enumerable: true, get: function () { return ceil_ts_1.ceil; } });
var comb_ts_1 = require("./comb.js");
Object.defineProperty(exports, "comb", { enumerable: true, get: function () { return comb_ts_1.comb; } });
var dist_ts_1 = require("./dist.js");
Object.defineProperty(exports, "dist", { enumerable: true, get: function () { return dist_ts_1.dist; } });
var exp_ts_1 = require("./exp.js");
Object.defineProperty(exports, "exp", { enumerable: true, get: function () { return exp_ts_1.exp; } });
var fabs_ts_1 = require("./fabs.js");
Object.defineProperty(exports, "fabs", { enumerable: true, get: function () { return fabs_ts_1.fabs; } });
var factorial_ts_1 = require("./factorial.js");
Object.defineProperty(exports, "factorial", { enumerable: true, get: function () { return factorial_ts_1.factorial; } });
var floor_ts_1 = require("./floor.js");
Object.defineProperty(exports, "floor", { enumerable: true, get: function () { return floor_ts_1.floor; } });
var fsum_ts_1 = require("./fsum.js");
Object.defineProperty(exports, "fsum", { enumerable: true, get: function () { return fsum_ts_1.fsum; } });
var gcd_ts_1 = require("./gcd.js");
Object.defineProperty(exports, "gcd", { enumerable: true, get: function () { return gcd_ts_1.gcd; } });
var isclose_ts_1 = require("./isclose.js");
Object.defineProperty(exports, "isclose", { enumerable: true, get: function () { return isclose_ts_1.isclose; } });
var isfinite_ts_1 = require("./isfinite.js");
Object.defineProperty(exports, "isfinite", { enumerable: true, get: function () { return isfinite_ts_1.isfinite; } });
var isinf_ts_1 = require("./isinf.js");
Object.defineProperty(exports, "isinf", { enumerable: true, get: function () { return isinf_ts_1.isinf; } });
var isnan_ts_1 = require("./isnan.js");
Object.defineProperty(exports, "isnan", { enumerable: true, get: function () { return isnan_ts_1.isnan; } });
var lcm_ts_1 = require("./lcm.js");
Object.defineProperty(exports, "lcm", { enumerable: true, get: function () { return lcm_ts_1.lcm; } });
var log_ts_1 = require("./log.js");
Object.defineProperty(exports, "log", { enumerable: true, get: function () { return log_ts_1.log; } });
var log10_ts_1 = require("./log10.js");
Object.defineProperty(exports, "log10", { enumerable: true, get: function () { return log10_ts_1.log10; } });
var log2_ts_1 = require("./log2.js");
Object.defineProperty(exports, "log2", { enumerable: true, get: function () { return log2_ts_1.log2; } });
var pow_ts_1 = require("./pow.js");
Object.defineProperty(exports, "pow", { enumerable: true, get: function () { return pow_ts_1.pow; } });
var sqrt_ts_1 = require("./sqrt.js");
Object.defineProperty(exports, "sqrt", { enumerable: true, get: function () { return sqrt_ts_1.sqrt; } });
var trunc_ts_1 = require("./trunc.js");
Object.defineProperty(exports, "trunc", { enumerable: true, get: function () { return trunc_ts_1.trunc; } });
