// locutus/golang/time/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.UnixMilli = exports.UnixMicro = exports.Unix = exports.Truncate = exports.Sub = exports.Round = exports.ParseInLocation = exports.ParseDuration = exports.Parse = exports.Format = exports.Equal = exports.Date = exports.Before = exports.After = exports.AddDate = exports.Add = void 0;
var Add_ts_1 = require("./Add.js");
Object.defineProperty(exports, "Add", { enumerable: true, get: function () { return Add_ts_1.Add; } });
var AddDate_ts_1 = require("./AddDate.js");
Object.defineProperty(exports, "AddDate", { enumerable: true, get: function () { return AddDate_ts_1.AddDate; } });
var After_ts_1 = require("./After.js");
Object.defineProperty(exports, "After", { enumerable: true, get: function () { return After_ts_1.After; } });
var Before_ts_1 = require("./Before.js");
Object.defineProperty(exports, "Before", { enumerable: true, get: function () { return Before_ts_1.Before; } });
var Date_ts_1 = require("./Date.js");
Object.defineProperty(exports, "Date", { enumerable: true, get: function () { return Date_ts_1.Date; } });
var Equal_ts_1 = require("./Equal.js");
Object.defineProperty(exports, "Equal", { enumerable: true, get: function () { return Equal_ts_1.Equal; } });
var Format_ts_1 = require("./Format.js");
Object.defineProperty(exports, "Format", { enumerable: true, get: function () { return Format_ts_1.Format; } });
var Parse_ts_1 = require("./Parse.js");
Object.defineProperty(exports, "Parse", { enumerable: true, get: function () { return Parse_ts_1.Parse; } });
var ParseDuration_ts_1 = require("./ParseDuration.js");
Object.defineProperty(exports, "ParseDuration", { enumerable: true, get: function () { return ParseDuration_ts_1.ParseDuration; } });
var ParseInLocation_ts_1 = require("./ParseInLocation.js");
Object.defineProperty(exports, "ParseInLocation", { enumerable: true, get: function () { return ParseInLocation_ts_1.ParseInLocation; } });
var Round_ts_1 = require("./Round.js");
Object.defineProperty(exports, "Round", { enumerable: true, get: function () { return Round_ts_1.Round; } });
var Sub_ts_1 = require("./Sub.js");
Object.defineProperty(exports, "Sub", { enumerable: true, get: function () { return Sub_ts_1.Sub; } });
var Truncate_ts_1 = require("./Truncate.js");
Object.defineProperty(exports, "Truncate", { enumerable: true, get: function () { return Truncate_ts_1.Truncate; } });
var Unix_ts_1 = require("./Unix.js");
Object.defineProperty(exports, "Unix", { enumerable: true, get: function () { return Unix_ts_1.Unix; } });
var UnixMicro_ts_1 = require("./UnixMicro.js");
Object.defineProperty(exports, "UnixMicro", { enumerable: true, get: function () { return UnixMicro_ts_1.UnixMicro; } });
var UnixMilli_ts_1 = require("./UnixMilli.js");
Object.defineProperty(exports, "UnixMilli", { enumerable: true, get: function () { return UnixMilli_ts_1.UnixMilli; } });
