// locutus/swift/String/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.trimmingCharacters = exports.split = exports.replacingOccurrences = exports.rangeOf = exports.padding = exports.hasSuffix = exports.hasPrefix = exports.componentsSeparatedBy = void 0;
var componentsSeparatedBy_ts_1 = require("./componentsSeparatedBy.js");
Object.defineProperty(exports, "componentsSeparatedBy", { enumerable: true, get: function () { return componentsSeparatedBy_ts_1.componentsSeparatedBy; } });
var hasPrefix_ts_1 = require("./hasPrefix.js");
Object.defineProperty(exports, "hasPrefix", { enumerable: true, get: function () { return hasPrefix_ts_1.hasPrefix; } });
var hasSuffix_ts_1 = require("./hasSuffix.js");
Object.defineProperty(exports, "hasSuffix", { enumerable: true, get: function () { return hasSuffix_ts_1.hasSuffix; } });
var padding_ts_1 = require("./padding.js");
Object.defineProperty(exports, "padding", { enumerable: true, get: function () { return padding_ts_1.padding; } });
var rangeOf_ts_1 = require("./rangeOf.js");
Object.defineProperty(exports, "rangeOf", { enumerable: true, get: function () { return rangeOf_ts_1.rangeOf; } });
var replacingOccurrences_ts_1 = require("./replacingOccurrences.js");
Object.defineProperty(exports, "replacingOccurrences", { enumerable: true, get: function () { return replacingOccurrences_ts_1.replacingOccurrences; } });
var split_ts_1 = require("./split.js");
Object.defineProperty(exports, "split", { enumerable: true, get: function () { return split_ts_1.split; } });
var trimmingCharacters_ts_1 = require("./trimmingCharacters.js");
Object.defineProperty(exports, "trimmingCharacters", { enumerable: true, get: function () { return trimmingCharacters_ts_1.trimmingCharacters; } });
